iceoryx2-bb/posix/src/config.rs

```diff
@@ -109,9 +112,9 @@ pub fn does_system_satisfy_posix_requirements(mode: ComplianceCheckMode) -> bool
     let mut is_compliant: bool = true;
 
     if mode == ComplianceCheckMode::Verbose {
-        println!("{HEADER_COLOR}system requirements check{COLOR_RESET}");
-        println!();
-        println!(" {HEADER_COLOR}minimum system{COLOR_RESET}");
+        info!("{HEADER_COLOR}system requirements check{COLOR_RESET}");
+        info!("");
+        info!(" {HEADER_COLOR}minimum system{COLOR_RESET}");
```

Comment:
nostd_println and all the other changes in this file

Reply:
For a porting effort it would make sense to have the output and if it is temporally the logger on `no_std`, this is totally fine. But it can be changed when necessary. So no action required.